src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisationTest.java

```diff
@@ -109,7 +112,18 @@ public void setup() {
 
     @Test
     void should_return_given_template_id() {
-        assertEquals(letterTemplateId, appellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisation.getTemplateId());
+        if (Objects.equals(asylumCase.read(REMISSION_DECISION, RemissionDecision.class), Optional.of(RemissionDecision.APPROVED))) {
+            assertEquals(approvedLetterTemplateId, appellantInternalLateRemissionPartiallyOrGrantedLetterPersonalisation.getTemplateId(asylumCase));
+        }
+        else if (Objects.equals(asylumCase.read(REMISSION_DECISION, RemissionDecision.class), Optional.of(RemissionDecision.PARTIALLY_APPROVED))) {
```

Comment:
This test doesn't go into any of these other if statements does it?
In the test set up we are setting the Remission Decision to Approved and never changing it, so it doesn't try the other branches.